Utilities/bootstrap

```diff
@@ -591,6 +604,7 @@ def build_swiftpm_with_cmake(args):
         "-DArgumentParser_DIR=" + os.path.join(args.swift_argument_parser_build_dir, "cmake/modules"),
         "-DSwiftDriver_DIR=" + os.path.join(args.swift_driver_build_dir, "cmake/modules"),
         "-DSwiftCrypto_DIR=" + os.path.join(args.swift_crypto_build_dir, "cmake/modules"),
+        "-DSwiftCollections_DIR=" + os.path.join(args.swift_collections_build_dir, "cmake/modules"),
```

Comment:
```suggestion
        "-DSwiftCollections_DIR=" + os.path.join(args.swift_collections_build_dir),
```

While this would be really nice to be uniform, it doesn't match the current build structure.